packages/rocketchat-lib/server/functions/notifications/mobile.js

```diff
@@ -46,6 +46,8 @@ export async function sendSinglePush({ room, message, userId, receiverUsername,
 			sender: message.u,
 			type: room.t,
 			name: room.name,
+			messageType: message.t,
```

Comment:
`message.t` will be _undefined_ most of the time. is that an issue?

Reply:
@sampaiodiego @cardoso Don't we have a default type for the message? That would be way better.

Reply:
@cardoso no, it will actually send `undefined` .. idk how that will sent to devices after parsed by push providers though.

@rafaelks there is no default type. messages are regular messages, except when they have a type =)

Reply:
No reason undefined should block this from being merged. This can be managed client-side.